backend/src/main/java/com/example/backend/model/ChatMessage.java

```diff
@@ -1,18 +1,19 @@
 package com.example.backend.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.MongoId;
 
-import java.time.LocalDateTime;
+import java.util.Date;
 
 @Document(collection = "chat_messages")
 public record ChatMessage(
     @MongoId
     String messageId,
     String senderId,
-    LocalDateTime timestamp,
-    String content,
-    ChatSession chatSession
+    String recipientId,
+    Date timestamp,
```

Comment:
eventuell DateTime verwenden um die Uhrzeit zu tracken?
```suggestion
    DateTime timestamp,
```

Reply:
Date funktioniert erstmal. Fürs Refactoring wird ein neues Issue angelegt